assets/src/edit-story/components/library/panes/media/media3p/media3pCategories.js

```diff
@@ -50,9 +52,22 @@ const CategoryPillContainer = styled.div`
 // Important: the visibily is 'inherit' when props.visible because otherwise
 // it gets shown even when the provider is not the selectedProvider!
 const ExpandButton = styled(ArrowDown)`
-  ${(props) => props.isExpanded && 'transform: matrix(1, 0, 0, -1, 0, 0);'};
-  visibility: ${(props) => (props.visible ? 'inherit' : 'hidden')};
+  display: flex;
+  position: absolute;
+  bottom: -12px;
+  background: ${({ theme }) => theme.colors.fg.gray16};
+  border-radius: 50px;
+  min-width: 32px;
+  min-height: 32px;
+  max-width: 32px;
+  max-height: 32px;
+  width: 32px;
+  height: 32px;
```

Comment:
nit: since width and height are the same as their min/max you could just use width/height and eliminate the min/max unless there is a use case I am not aware of here :)

Reply:
Cleaned it up in 6d47912c4.

The `max-height` is necessary to overrule the `max-height: 30px` from [the base button class](https://github.com/google/web-stories-wp/blob/main/assets/src/edit-story/components/button/index.js#L51). I think @wassgha just added them all to be safe in his frustration over the button insisting on having the wrong size 😂

Reply:
ah, that's good context! 

Since this is foundational work for the new design system, is it worth considering eliminating the max-height on the base if it's going to be consistently overridden?